### DISCUSSION TITLE
Prefers Colour Scheme: Dark

### DISCUSSION THREAD

**Discussion by Quin452** [konstaui/konsta#76] [Q&A]
So, I've installed Konsta on a Vue project, and it basically wants to default to dark theme.

How do I change this?

Literally, the template is simply:
```
<template>
    <k-app theme="material">
        <router-view/>
    </k-app>
</template>
<script>
import { kApp } from 'konsta/vue';

export default {
    components: {
        kApp
    },
};
</script>
```
And yet, Dark Theme is being used (which I do not want).

**Comment by nolimits4web** [comment]
Check tailwind docs and how dark theme should be configured there

**Reply by Quin452** [reply]
Thanks.  It wasn't something set in Tailwind, but instead pulled on my browsers theme which was set to Dark Mode (Tailwind does pull on `prefers-color-scheme`).
There is also an option in the OS too (just in-case someone comes across this issue too).

Honestly, I didn't even know it "was a thing".
